gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/LoginImageController.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 OpsMX, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.gate.controllers;
+
+import com.netflix.spinnaker.gate.config.GateLoginImageConfigProperties;
+import io.swagger.annotations.ApiOperation;
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/image")
+public class LoginImageController {
+
+  @Autowired GateLoginImageConfigProperties gateLoginLogoConfigProperties;
```

Comment:
Avoid the use of `@Autowired` properties. Use constructor injection instead.

Reply:
Added design document for this PR https://github.com/spinnaker/governance/pull/115